subsys/net/lib/nrf_cloud/src/nrf_cloud_fsm.c

```diff
@@ -522,6 +522,14 @@ static int dc_rx_data_handler(const struct nct_evt *nct_evt)
 		.topic = nct_evt->param.dc->topic,
 	};
 
+	if(nrf_cloud_detect_disconnection_request(nct_evt->param.dc->data.ptr)) {
+		int err = nrf_cloud_disconnect();
+		if (err < 0) {
+			LOG_ERR("Failed to disconnect from nRF Cloud in response to disconnection request, error: %d", err);
+		}
```

Comment:
It would be good to add a LOG_DBG() here to indicate that the disconnect was requested from the cloud side.

Reply:
Consider it done!

Reply:
I had to shorten that debug line, too, by the way, for compliance reasons, in case that alters your standpoint at all. (I doubt that it does)

Reply:
It happens.  You can split the string onto the next line, e.g.:
```
    LOG_DBG("here is a really long "
                     "string.  It is absolutely huge.");

Reply:
Done!